irohad/network/CMakeLists.txt

```diff
@@ -40,3 +40,11 @@ add_library(ordering_gate_common
 target_link_libraries(ordering_gate_common
     boost
     )
+
+add_library(async_grpc_client INTERFACE
+    # impl/async_grpc_client.hpp
```

Comment:
is it left purportedly?

Reply:
It is because it is an interface target.

Reply:
I'm not insisting, but I think that if someone fails to compile something because this target is not a dependee of some unit using this header, he will more likely find a reference to the needed target in that header file, than the reference to that header in this target (as he probably does not even suppose that he is missing a target dependency).